### DISCUSSION TITLE
Creating a new Service

### DISCUSSION THREAD

**Discussion by M-JANDAR12** [HewlettPackard/squest#613] [Q&A] [answered]
I'm trying to test the Demo of creating a file, I create a new service and an operation "Create". I didn't add any fileds in the extra vars filed. The job template has one survey item which is "file_name". when ordering the Service I am asked to give the file name. The problem is that Tower is not getting the "squest" variable, I see in the job description no extra variables other than file_name. but in squest backend logs it is shown that it was sent
Backend Output:
![image](https://github.com/HewlettPackard/squest/assets/74101571/45ec8eb4-2b69-404a-a316-55211b7125b9)

AWX Job Description:
![image](https://github.com/HewlettPackard/squest/assets/74101571/c42d0e80-50d9-4bbe-b492-fd62cbc3a5fe)

Am I doing something wrong?

**Comment by a-belhadj** [comment, marked as answer]
Hello,
![image](https://github.com/HewlettPackard/squest/assets/82872898/cf4899a8-d65e-4a6b-980d-a134b2283ce1)


To see it in Squest go to Tower/AWX on Squest and click on your job templates then on the red cross
![image](https://github.com/HewlettPackard/squest/assets/82872898/675fdd8c-a720-49fd-8398-523442688cb3)
![image](https://github.com/HewlettPackard/squest/assets/82872898/b81ed72c-ad11-4a0f-b19c-e761fae93069)


Anthony

**Reply by M-JANDAR12** [reply]
It worked
Thanks
